simplivity/resources/external_stores.py

```diff
@@ -111,6 +111,29 @@ def register_external_store(self, management_ip, name, cluster, username, passwo
 
         return self.get_by_name(name)
 
+    def unregister_external_store(self, name, cluster, timeout=-1):
```

Comment:
Should we move this to the ExternalStore class as external store name is mandatory?
so that we can call this method on ExternalStore object, like obj.unregister(cluster)

Reply:
What's your opinion on this? Can we move this method to the ExternalStore class?

Reply:
Updated the code as suggested